python-sdk/pyproject.toml

```diff
@@ -17,7 +17,7 @@ license = {file = "LICENSE"}
 
 requires-python = ">=3.7"
 dependencies = [
-    "apache-airflow>=2.0",
+    "apache-airflow>=2.4.0rc1",
```

Comment:
Why are we bumping the **minimum** version?

Reply:
Ah sorry, this is only for testing, right @phanikumv?

Reply:
this is to test the code on 2.4.0b1. This PR is just to check whether all the CI unit tests run fine or not. We wont merge it

Reply:
thats right @feluelle